Assets/MRTK/SDK/Experimental/RiggedHandVisualizer/RiggedHandVisualizer.cs

```diff
@@ -0,0 +1,254 @@
+                joints[TrackedHandJoint.ThumbProximalJoint] = ThumbMetacarpal.GetChild(0);
+                joints[TrackedHandJoint.ThumbDistalJoint] = ThumbMetacarpal.GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.ThumbTip] = ThumbMetacarpal.GetChild(0).GetChild(0).GetChild(0);
+            }
+            if (IndexMetacarpal)
+            {
+                joints[TrackedHandJoint.IndexMetacarpal] = IndexMetacarpal;
+                joints[TrackedHandJoint.IndexKnuckle] = IndexMetacarpal.GetChild(0);
+                joints[TrackedHandJoint.IndexMiddleJoint] = IndexMetacarpal.GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.IndexDistalJoint] = IndexMetacarpal.GetChild(0).GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.IndexTip] = IndexMetacarpal.GetChild(0).GetChild(0).GetChild(0).GetChild(0);
+            }
+            if (MiddleMetacarpal)
+            {
+                joints[TrackedHandJoint.MiddleMetacarpal] = MiddleMetacarpal;
+                joints[TrackedHandJoint.MiddleKnuckle] = MiddleMetacarpal.GetChild(0);
+                joints[TrackedHandJoint.MiddleMiddleJoint] = MiddleMetacarpal.GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.MiddleDistalJoint] = MiddleMetacarpal.GetChild(0).GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.MiddleTip] = MiddleMetacarpal.GetChild(0).GetChild(0).GetChild(0).GetChild(0);
+            }
+            if (RingMetacarpal)
+            {
+                joints[TrackedHandJoint.RingMetacarpal] = RingMetacarpal;
+                joints[TrackedHandJoint.RingKnuckle] = RingMetacarpal.GetChild(0);
+                joints[TrackedHandJoint.RingMiddleJoint] = RingMetacarpal.GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.RingDistalJoint] = RingMetacarpal.GetChild(0).GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.RingTip] = RingMetacarpal.GetChild(0).GetChild(0).GetChild(0).GetChild(0);
+            }
+            if (PinkyMetacarpal)
+            {
+                joints[TrackedHandJoint.PinkyMetacarpal] = PinkyMetacarpal;
+                joints[TrackedHandJoint.PinkyKnuckle] = PinkyMetacarpal.GetChild(0);
+                joints[TrackedHandJoint.PinkyMiddleJoint] = PinkyMetacarpal.GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.PinkyDistalJoint] = PinkyMetacarpal.GetChild(0).GetChild(0).GetChild(0);
+                joints[TrackedHandJoint.PinkyTip] = PinkyMetacarpal.GetChild(0).GetChild(0).GetChild(0).GetChild(0);
+            }
+
+            // Precalculated values for LeapMotion testhand
+            FingerTipLengths[TrackedHandJoint.ThumbTip] = 0.02167f;
+            FingerTipLengths[TrackedHandJoint.IndexTip] = 0.01582f;
+            FingerTipLengths[TrackedHandJoint.MiddleTip] = 0.0174f;
+            FingerTipLengths[TrackedHandJoint.RingTip] = 0.0173f;
+            FingerTipLengths[TrackedHandJoint.PinkyTip] = 0.01596f;
```

Comment:
It's best to avoid seemingly magic numbers, it's helpful to store the pre-calculated values in another var with a descriptive name.